applications/fishing-map/src/features/workspace/workspace.mock.ts

```diff
@@ -1,27 +1,55 @@
 /* eslint-disable @typescript-eslint/camelcase */
 import { Workspace } from '@globalfishingwatch/dataviews-client'
+import { Generators } from '@globalfishingwatch/layer-composer'
 
 const workspace: Workspace = {
   id: 1,
   name: 'Default public Fishing Map workspace',
   description: '',
   viewport: {
-    latitude: 11,
-    longitude: -72,
-    zoom: 3,
+    zoom: 4,
+    latitude: 42,
+    longitude: -9,
   },
   start: '2018-01-01T00:00:00.000Z',
   end: '2019-12-31T00:00:00.000Z',
   dataviews: [
+    {
+      id: 10,
+      name: 'background',
+      description: '',
+      config: {
+        id: 'background',
+        type: Generators.Type.Background,
+        color: '#00265c',
+      },
+    },
+    {
+      id: 11,
+      name: 'basemap',
+      description: '',
+      config: {
+        id: 'basemap',
+        type: Generators.Type.Basemap,
+        basemap: Generators.BasemapType.Default,
+      },
+    },
     {
       id: 1,
       name: 'Apparent fishing effort',
       description:
         'We use data about a vessel’s identity, type, location, speed, direction and more that is broadcast using the Automatic Identification System (AIS) and collected via satellites and terrestrial receivers. We analyze AIS data collected from vessels that our research has identified as known or possible commercial fishing vessels, and apply a fishing detection algorithm to determine “apparent fishing activity” based on changes in vessel speed and direction. The algorithm classifies each AIS broadcast data point for these vessels as either apparently fishing or not fishing and shows the former on our fishing activity heat map.',
       config: {
-        type: 'HEATMAP',
+        type: Generators.Type.HeatmapAnimated,
         color: '#00FFBC',
         colorRamp: 'teal',
+        sublayers: [{ id: 'fishing', colorRamp: 'teal', datasets: ['fishing_v4'] }],
+        combinationMode: 'compare',
+        debug: false,
+        debugLabels: false,
+        geomType: 'gridded',
+        tilesAPI: `${process.env.REACT_APP_API_GATEWAY}/v1/4wings`,
+        interactive: true,
```

Comment:
I would remove non-mandatory generator params where we don't need another value than the default: debug, debugLabels, geomType, interactive

also `colorRamp` is deprecated.